ansible/group_vars/all

```diff
@@ -62,6 +62,9 @@ runtimesManifest:
       default: true
       image:
         name: "swift3action"
+    - kind: "swift:3.1.1"
```

Comment:
OK we can't deprecate, it doesn't mean what I thought, deprecate:true is that we don't accept new action to be created with this kind.